lib/msal-core/src/Configuration.ts

```diff
@@ -117,6 +118,7 @@ const DEFAULT_AUTH_OPTIONS: AuthOptions = {
     clientId: "",
     authority: null,
     validateAuthority: true,
+    authorityMetadata: "",
```

Comment:
Do we want this to be null? If not passed in we dont want this to evaluate to truthy

Reply:
Empty string is falsey.